### PR TITLE
Apply compat changes from latest Pekko

### PR DESCRIPTION
**Pull request by nvollmar**

### CODE REVIEW

File: core/src/main/scala/org/apache/pekko/persistence/cassandra/query/EventsByPersistenceIdStage.scala
Comment (mdedetrich) [inline core/src/main/scala/org/apache/pekko/persistence/cassandra/query/EventsByPersistenceIdStage.scala:19]: Can we keep the `pekko` prefix here rather than using `org.apache.pekko`. While there is an argument to use the FQCN the community needs to decide on this and it will likely be done as part of a scalafix integration.

File: project/Dependencies.scala
Comment (mdedetrich) [inline project/Dependencies.scala:26]: So I am currently working on pekko-connectors (see https://github.com/apache/incubator-pekko-connectors/pull/101) and there are some blocking upstream changes needed to get it to work.

Due to dependency management hell, it would be ideal to release pekko-connectors once that PR is merged and use that version here.

**Review by mdedetrich — CHANGES_REQUESTED** [review]
Nice work, there is one change regarding imports. As mentioned in the review, this PR should be updated once pekko-connectors is released with the newest pekko upstream changes.

File: project/Dependencies.scala
Comment (nvollmar) [reply project/Dependencies.scala:26]: yep, I've to wait for a new pekko build with those changes

File: core/src/main/scala/org/apache/pekko/persistence/cassandra/query/EventsByPersistenceIdStage.scala
Comment (nvollmar) [reply core/src/main/scala/org/apache/pekko/persistence/cassandra/query/EventsByPersistenceIdStage.scala:19]: will revert, organize import automatically broke that

File: project/Dependencies.scala
Comment (mdedetrich) [inline project/Dependencies.scala:26]: @nvollmar 0.0.0+85-a82f3c3c-SNAPSHOT was just released

Comment (nvollmar) [reply project/Dependencies.scala:26]: thanks, will upgrade and re-test

File: project/Dependencies.scala
Comment (mdedetrich) [inline project/Dependencies.scala:52]: So I suspect that you originally added this because you had mismatched snapshot versions, if this is the case I suspect that if you remove this dependency it should still work/pass.

**Review by mdedetrich — APPROVED** [review]
lgtm, thanks for pushing this through. I made one comment at https://github.com/apache/incubator-pekko-persistence-cassandra/pull/46/files#r1190894767 if you want to verify but it shouldn't block the PR.

File: project/Dependencies.scala
Comment (nvollmar) [reply project/Dependencies.scala:52]: Yes, but since the project relies heavily on streams it would make sense to me to keep the explicit dependency.

Comment (mdedetrich) [reply project/Dependencies.scala:52]: Fair enough 👍

**Comment by mdedetrich** [comment]
Tests passed @nvollmar you want to merge?

**Comment by mdedetrich** [comment]
Publish action has been manually triggered https://github.com/apache/incubator-pekko-persistence-cassandra/actions/runs/4946991536

**Comment by mdedetrich** [comment]
New version is `1079-c30f82ae-SNAPSHOT`

**Pull request merged**
